commit e3f18148ffe2b731febc38474f39932aef34c650
Author: halton <halton@judo.(none)>
Date:   Wed Oct 22 15:01:42 2008 +0800

    Fix zero-sized struct build problem with sun cc. Bug #18149

diff --git a/src/ck-log-event.h b/src/ck-log-event.h
index 64dec58..a54a34e 100644
--- a/src/ck-log-event.h
+++ b/src/ck-log-event.h
@@ -45,18 +45,22 @@ typedef enum
 
 typedef struct
 {
+   gboolean foo;
 } CkLogNoneEvent;
 
 typedef struct
 {
+   gboolean foo;
 } CkLogSystemStopEvent;
 
 typedef struct
 {
+   gboolean foo;
 } CkLogSystemRestartEvent;
 
 typedef struct
 {
+   gboolean foo;
 } CkLogSystemStartEvent;
 
 typedef struct
